.github/workflows/build-test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Is there any harm in removing this, and running on the `pull_request` event for any branch? CI is useful sometimes for making changes to feature branches

```suggestion
```